tests/clients/test_ngsi_v2_timeseries.py

```diff
@@ -119,6 +119,39 @@ def test_input_endpoints(self) -> None:
     @clean_test(fiware_service=settings.FIWARE_SERVICE,
                 fiware_servicepath=settings.FIWARE_SERVICEPATH,
                 ql_url=settings.QL_URL)
+
```

Comment:
@StWiemann please remove blankline. The decorator must be directly above the function definition

Reply:
@StWiemann I have done this now